packages/trader/src/App/Components/Form/CompositeCalendar/composite-calendar-mobile.jsx

```diff
@@ -112,7 +112,23 @@ const CompositeCalendarMobile = React.memo(
             }
 
             if (key === 'to') {
-                setTo(e.target?.value ? toMoment(e.target.value).format('DD MMM YYYY') : '');
+                // Handling future dates input for IOS devices
+                const value = e.target?.value ? toMoment(e.target.value).format('DD MMM YYYY') : '';
+
+                setTo(value);
+
+                if (value) {
+                    const upcoming_days = parseInt(daysFromTodayTo(value));
+                    if (!isNaN(upcoming_days)) {
+                        setTo('');
+
+                        const today = toMoment().format('DD MMM YYYY');
```

Comment:
Nitpick: we can move this declaration on line 123, it's better to have all declarations on the top of their scope